billing-ui/src/utils/transforms.js

```diff
@@ -33,6 +33,8 @@ export const collectionFlattener = (item, parnetKey, stem = '') => {
         : { [newStem]: item };
 };
 
-export const customNumberSort = (a, b, order, field) => ((order === 'desc')
-    ? a[field] - b[field]
-    : b[field] - a[field]);
+export const customNumberSort = (a, b, order) => (
```

Comment:
The latest Bootstrap Tablas pass the value directly, not the row + fieldName anymore.